packages/falso/src/lib/rand-array.ts

```diff
@@ -0,0 +1,22 @@
+import { fake, FakeOptions } from './core/core';
+
+/**
+ * Get a random item from array.
+ *
+ * @category general
+ *
+ * @example
+ *
+ * randArray([1, 2, 3])
+ *
+ * @example
+ *
+ * randArray([1, 2, 3], { length: 10 })
+ *
+ */
+export function randArray<T, Options extends FakeOptions = never>(
```

Comment:
This name sounds like a function that returns a random array, which isn't always the case.

We can change the `rand` function to `_rand` and leave this as `rand`. You're welcome to suggest a different name.

Reply:
I'm not a big fan of _function for internal function. 

I named it randElt if it's ok for you

Reply:
What is elt?

Reply:
random element

Reply:
Let's call it randElement